templates/repo/release/new.tmpl

```diff
@@ -38,6 +38,9 @@
 								</div>
 							</div>
 						</div>
+						<div id="tag-warning" class="gt-dib gt-hidden" data-commit-url-stub="{{.RepoLink}}/commit">
+							{{svg "octicon-alert-fill" 16}} Existing tag referencing <span class="gt-px-3">{{svg "octicon-git-commit" 16}} <a target="_blank" rel="noopener noreferrer" class="tag-warning-detail"></a></span>
+						</div>
```

Comment:
I think this warning is good enough. 

We can make it more prominent, then the modals could (should) be removed.

The problem is that:

* There are too many duplicate warnings when using the modal.
* There are too many problems in the modal related code (so remove the modals, then we do not need to spend time on it)